src/test/java/ay2122s1_cs2103t_w16_2/btbb/testutil/OrderBuilder.java

```diff
@@ -8,14 +11,23 @@
  */
 public class OrderBuilder {
     private static final String DEFAULT_PHONE = "85355255";
+    private static final String DEFAULT_NAME = "Amy Bee";
```

Comment:
Same thing, add `CLIENT`. (This will prevent conflict with recipe variables later. When we have things like `RECIPE_NAME`)